index.js

```diff
@@ -4,8 +4,15 @@ var request = require('request')
   , qs = require('querystring')
   , jsonreq = request.defaults({json: true})
 
-function makeError (err, resp) {
-  var errObject = new Error(resp.statusCode + ' ' + err.reason)
+function makeError (err, resp, meta) {
+  var errMessage = resp.statusCode + ' ' + err.reason
+    , errObject
+
+  if (meta) {
+    errMessage += JSON.stringify(meta)
```

Comment:
i think you should add a newline before the JSON data, and pretty print it. this is gonna be hard to read.

`errMessage += '\n' + JSON.stringify(meta, null, 2)`